mod/deckorate_utility/set/all/wikirate/icons.rb

```diff
@@ -38,7 +38,8 @@
   board: :build,
   reference: :attribution,
   history: :history,
-  clipboard: :content_copy
+  clipboard: :content_copy,
+  close: :close
```

Comment:
close is actually already defined in vendor/decko/mod/bootstrap/set/all/bootstrap/icon.rb

(it's used in decko, not just wikirate)